loadtest/user/userentity/actions.go

```diff
@@ -157,6 +157,20 @@ func (ue *UserEntity) CreateDirectChannel(otherUserId string) (string, error) {
 	return channel.Id, nil
 }
 
+func (ue *UserEntity) AddChannelMember(channelId, userId string) (*model.ChannelMember, error) {
```

Comment:
Between return a custom struct or a map an return the `model.ChannelMember` structs, I decided for the latter

Reply:
The problem with this is that we don't want to return a pointer to the data we are storing cause that makes it potentially mutable by the user of this function. One solution would be to dereference and return a copy of `model.ChannelMember` but I'd avoid that for now cause it brings other issues and it wouldn't be consistent with other functions. I suggest we just return the error for now and revisit this later when we need it.

Reply:
Ok! :)

Reply:
Changed to return only error by now